### DISCUSSION TITLE
Metal 2 support

### DISCUSSION THREAD

**Discussion by rajkr09** [google/filament#4188] [General]
Does the Filament support Metal 2 API (the recent api), if not, do you have plans to support it in future.. I'm stuck between choosing either "The Forge" or "Filament" because of this feature.

**Comment by romainguy** [comment]
@bejado

**Reply by bejado** [reply]
We support iOS 11.0 and above so we're somewhat limited in the Metal APIs we can use, but yes, we do use some Metal 2 features and I have plans to add support for more (like argument buffers).

**Reply by rajkr09** [reply]
I was thinking of GPU based Metal Acceleration and Intersection structures introduced in Metal 2. I don't know how much it will be of performance boost but
thank you for the information.
